internals/models.py

```diff
@@ -946,18 +949,14 @@ def new_crbug_url(self):
       params.append('cc=' + ','.join(self.owner))
     return url + '?' + '&'.join(params)
 
-  @classmethod
-  def _from_pb(cls, pb, set_key=True, ent=None, key=None):
+  def stash_values(self):
 
     # Stash existing values when entity is created so we can diff property
     # values later in put() to know what's changed. https://stackoverflow.com/a/41344898
-    entity = super(Feature, cls)._from_pb(pb, set_key=set_key, ent=ent, key=key)
     
-    for prop_name, prop in entity._properties.iteritems():
-      old_val = getattr(entity, prop_name, None)
-      setattr(entity, '_old_' + prop_name, old_val)
-
-    return entity
+    for prop_name, prop in self._properties.iteritems():
+      old_val = getattr(self, prop_name, None)
+      setattr(self, '_old_' + prop_name, old_val)
 
```

Comment:
We can use this method for stashing the old values. It seems to work fine. 

FIY, The diffs still show three extra fields that are not useful to the user. These fields belong to None vs False case. Hence, merging main branch in this PR, will solve this problem completely.

Reply:
Sounds good.